stdpopsim/catalog/arabidopsis_thaliana.py

```diff
@@ -66,15 +67,16 @@
 
 _gm = stdpopsim.GeneticMap(
     species=_species,
-    name="Salome2012_TAIR7",
+    id="Salome2012_TAIR7",
+    description="Salome map FIXME",
```

Comment:
not sure why this says FIXME at this point?

Reply:
"Salome map" is pretty undescriptive..